README.md

```diff
@@ -133,7 +142,7 @@ and written to corresponding SP memory.
 
 validtepair*(...) checks of each VM pair would be consistent with a track of Pt > 2 GeV.
 
-For displaced tracking, function validtedpair*(...) used instead, which checks if the VM pair is consistent with |d0| < 3cm for all possible abs(q/Pt) < 1/2.
+For displaced tracking, function validtedpair*(...) used instead, which requires the VM pair to be consistent with |d0| < 3cm for at least one of abs(q/Pt) = 1/2 or -1/2.
```

Comment:
Remove abs() in this line. Everything else looks good.

Reply:
I agree with Derek's comment and have committed the corresponding fix.